app/uk/gov/hmrc/pillar2externalteststub/models/NilReturnSuccess.scala

```diff
@@ -17,15 +17,16 @@
 package uk.gov.hmrc.pillar2externalteststub.models
 
 import play.api.libs.json.{Json, OFormat}
-import uk.gov.hmrc.pillar2externalteststub.models.uktr.UktrSubmission
+import uk.gov.hmrc.pillar2externalteststub.models.subscription.SubscriptionResponse
+import uk.gov.hmrc.pillar2externalteststub.models.uktr.UKTRSubmission
 
-case class NilReturnSuccess(processingDate: String, message: String)
+case class NilReturnSuccess(processingDate: String, message: String) extends SubscriptionResponse
```

Comment:
This builds on the presumption that NilReturnSuccess is a Subscription response as has been followed prior to this PR.

Reply:
Yes, that makes sense - as per the case statement in object SubscriptionResponse - this is the response returned when a user asks for the details for a previosuly submitted Nil Return.